docs/documentation/aggregates/overview.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Overview
+---
+
+## Basic Usage
+
+ParadeDB supports aggregates through plain SQL functions like `COUNT`, `SUM`, and `AVG`. For instance, the following query finds the total number of search results for a full text query.
+
+```sql
+SELECT COUNT(*) FROM mock_items
+WHERE description @@@ 'shoes';
+```
+
+## Fast Aggregates
+
+<Info>
+  Fast aggregates are an enterprise-only feature. [Contact us](mailto:sales@paradedb.com) for access.
+</Info>
+
+The performance of aggregates over very large result sets (i.e. millions of rows) is a well-known problem in Postgres.
+
+In ParadeDB enterprise, aggregates over full text search results are drastically accelerated by [fast fields](/documentation/indexing/fast_fields).
+Fast fields are stored in a column-oriented fashion, which makes them much quicker to read and process.
+
+You can check which fields are indexed as fast by running `paradedb.schema`.
+
+```sql
+SELECT name, fast FROM paradedb.schema('search_idx') ORDER BY fast;
+```
+
+<Accordion title="Expected Response">
+```csv
+     name     | fast
+--------------+------
+ metadata     | f
+ description  | f
+ category     | f
+ id           | t
+ in_stock     | t
+ rating       | t
+ weight_range | t
+ created_at   | t
+ ctid         | t
+(9 rows)
+```
+</Accordion>
+
+In order for a query to be accelerated, **all** of the returned columns must be fast fields. For instance, the following query is accelerated because both `rating`
+and `id` are fast fields:
+
+```sql
+SELECT rating, COUNT(id) FROM mock_items
+WHERE description @@@ 'shoes'
+GROUP BY rating LIMIT 5;
+```
+
+You can verify if a query will be accelerated by running `EXPLAIN`. Accelerated queries will show either a `Custom Scan` or `Parallel Index Only Scan` in the query plan.
+
+```sql
+EXPLAIN SELECT rating, COUNT(id) FROM mock_items
```

Comment:
```suggestion
EXPLAIN SELECT rating, COUNT(*) FROM mock_items
```

Reply:
I purposely did `COUNT(id)` because it's more clear to the user that `id` and `rating` are both fast. If I do `COUNT(*)` then I need to explain here that `*` is rewritten to `id` which feels like information overload, what do you think

Reply:
What the query is asking for is the count of all matching docs, by rating.  So to me, `COUNT(*)` is the intuitive construction.

If you think explaining `COUNT(*)` will end up as information overload, that's fine.  Maybe I'm an outlier, but `COUNT(<field>)` makes me scratch my head for a minute.

Reply:
Ok, let me think some more about it